vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.cpp

```diff
@@ -174,121 +225,76 @@ void WindowsModalHostComponentView::RegisterWndClass() noexcept {
   registered = true;
 }
 
+winrt::Microsoft::ReactNative::Composition::Experimental::IVisual
+WindowsModalHostComponentView::VisualToMountChildrenInto() noexcept {
+  return m_reactNativeIsland
+      .as<winrt::Microsoft::ReactNative::Composition::Experimental::IInternalCompositionRootView>()
+      .InternalRootVisual();
+}
+
+// childComponentView - reference to the child component view
+// index - the position in which the childComponentView should be mounted
 void WindowsModalHostComponentView::MountChildComponentView(
     const winrt::Microsoft::ReactNative::ComponentView &childComponentView,
     uint32_t index) noexcept {
-  // Disabled due to partial Modal implementation. Tracking re-enablement with task list here:
-  // https://github.com/microsoft/react-native-windows/issues/11157 assert(false);
+  EnsureModalCreated();
   base_type::MountChildComponentView(childComponentView, index);
 }
 
 void WindowsModalHostComponentView::UnmountChildComponentView(
     const winrt::Microsoft::ReactNative::ComponentView &childComponentView,
     uint32_t index) noexcept {
-  // Disabled due to partial Modal implementation.Tracking re-enablement with task list here : https : //
-  // github.com/microsoft/react-native-windows/issues/11157 assert(false);
   base_type::UnmountChildComponentView(childComponentView, index);
 }
 
-void WindowsModalHostComponentView::HandleCommand(
```

Comment:
Do we no longer need this?

Reply:
No, I had that there originally to see the order of events in Modal but there's nothing different here than it's parent function :)